src/resources/views/components/dropdown/dropdown.blade.php

```diff
@@ -31,10 +33,9 @@
              x-transition:leave="transition ease-in duration-75"
```

Comment:
Please, try to add an animation using the `transition-all ease-in-out duration-200`